doc/getting_started/beginner_tutorial.rst

```diff
@@ -1,28 +1,24 @@
 Beginner Tutorial
 =================
-Welcome to the beginner tutorial for Pachyderm. If you've already got Pachyderm installed, this guide should take about 15 minutes and you'll be introduced to the basic concepts of Pachyderm.
+Welcome to the beginner tutorial for Pachyderm! If you've already got Pachyderm installed, this guide should take about 15 minut!s, and it will introduce you to the basic concepts of Pachyderm.
 
 Image processing with OpenCV
 ----------------------------
 
-In this guide we're going to create a Pachyderm pipeline to do some simple `edge detection <https://en.wikipedia.org/wiki/Edge_detection>`_ on a few images. Thanks to Pachyderm's processing system, we'll be able to run the pipeline in a distributed, streaming fashion. As new data is
-added, the pipeline will automatically process it and output the results.
+In this guide we're going to create a Pachyderm pipeline to do some simple `edge detection <https://en.wikipedia.org/wiki/Edge_detection>`_ on a few images. Thanks to Pachyderm's processing system, we'll be able to run the pipeline in a distributed, streaming fashion. As new data is added, the pipeline will automatically process it and output the results.
 
-If you hit any errors not covered in this guide, check our :doc:`troubleshooting` docs for common errors, submit an issue on `GitHub <https://github.com/pachyderm/pachyderm>`_, join our `users channel on Slack <http://slack.pachyderm.io>`_, or email us at `support@pachyderm.io <mailto:support@pachyderm.io>`_ and we can help you right away.
+If you hit any errors not covered in this guide, get help in our `public commity Slack <http://slack.pachyderm.io>`_, submit an issue on `GitHub <https://github.com/pachyderm/pachyderm>`_, or email us at `support@pachyderm.io <mailto:support@pachyderm.io>`_. We are more than happy to help!
```

Comment:
Are the troubleshooting docs not actually helpful wrt to the tutorial?

Reply:
@JoeyZwicker Not anymore.  They actually refer to fruit stand still and `grep`.